swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageStatusConsumer.java

```diff
@@ -81,7 +81,7 @@ private OffsetDateTime nextTick() {
   }
 
   private boolean needsRetry(BillableUsageAggregate billableUsageAggregate) {
-    return billableUsageAggregate.getStatus() == FAILED
+    return billableUsageAggregate.getStatus() == RETRYABLE
```

Comment:
should the status "RETRYABLE" being enough to perform the retry (without looking the other error code conditions)?
Also, we need to add a migration script for the failed+error code conditions to be moved to RETRYABLE.

Reply:
IMO I think it makes more sense to remove the conditions on error_code and allow the producers to decide what is retry-able without having to worry about updating logic in both billable-usage and the producer but ok with either.

Reply:
Sure. I am okay either ways since there are pros and cons either ways. I will remove that errorcode check and I also see I forgot to add the same piece of code for Azure. Will make the changes.

Reply:
Done